orm-rules-tests/rules-test/custom-globals/max-timeout-server-0/rules/test_max_timeout_server.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+schema_version: 1
+
+rules:
+  - description: 'Test slow backend with custom server timeout set higher than max_timeout_server'
```

Comment:
The custom server timeout is the same as the max, that is infinite?

Reply:
I'll rephrase that into something descriptive.